earth_enterprise/src/third_party/libcurl/SConscript

```diff
@@ -17,7 +18,7 @@
 
 
 Import('third_party_env')
-libcurl_version = 'curl-7.45.0'
+libcurl_version = 'curl-7.74.0'
```

Comment:
We should update `README.google` in this same directory because it also has the curl version number in it.